api_tests/nodes/views/test_node_embeds.py

```diff
@@ -1,100 +1,129 @@
-from nose.tools import *  # flake8: noqa
+import pytest
 import functools
 
 from framework.auth.core import Auth
-
 from api.base.settings.defaults import API_BASE
 from tests.base import ApiTestCase
 from osf_tests.factories import (
     ProjectFactory,
     AuthUserFactory
 )
 
-class TestNodeEmbeds(ApiTestCase):
+@pytest.fixture()
+def user():
+    return AuthUserFactory()
+
+@pytest.mark.django_db
+class TestNodeEmbeds:
+
+    @pytest.fixture()
+    def contrib_one(self):
+        return AuthUserFactory()
+
+    @pytest.fixture()
+    def contrib_two(self):
+        return AuthUserFactory()
+
+    @pytest.fixture()
+    def contribs(self, contrib_one, contrib_two):
+        return [contrib_one, contrib_two]
+
+    @pytest.fixture()
+    def auth(self, user):
+        return Auth(user)
+
+    @pytest.fixture()
+    def make_public_node(self, user):
+        return functools.partial(ProjectFactory, is_public=False, creator=user)
+
+    @pytest.fixture()
+    def root_node(self, auth, contrib_one, contrib_two, make_public_node):
+        root_node = make_public_node()
+        root_node.add_contributor(contrib_one, ['read', 'write'], auth=auth, save=True)
+        root_node.add_contributor(contrib_two, ['read', 'write'], auth=auth, save=True)
+        return root_node
 
-    def setUp(self):
-        super(TestNodeEmbeds, self).setUp()
+    @pytest.fixture()
+    def child_one(self, auth, contrib_one, contrib_two, make_public_node, root_node):
+        child_one = make_public_node(parent=root_node)
+        child_one.add_contributor(contrib_one, ['read', 'write'], auth=auth, save=True)
+        child_one.add_contributor(contrib_two, ['read', 'write'], auth=auth, save=True)
```

Comment:
My first instinct is to say that one save would be faster, but I don't think it would be easy to accurately test that and I think the difference would likely be pretty marginal